cmd/eksctl-anywhere/cmd/get.go

```diff
@@ -11,3 +25,49 @@ var getCmd = &cobra.Command{
 func init() {
 	rootCmd.AddCommand(getCmd)
 }
+
+func preRunGetPackages(cmd *cobra.Command, args []string) error {
+	if !features.IsActive(features.CuratedPackagesSupport()) {
+		return fmt.Errorf("this command is currently not supported")
+	}
+	cmd.Flags().VisitAll(func(flag *pflag.Flag) {
+		if err := viper.BindPFlag(flag.Name, flag); err != nil {
+			log.Fatalf("Error initializing flags: %v", err)
+		}
+	})
+	return nil
+}
+
+func getResources(ctx context.Context, resourceType string, output string, args []string) error {
+	kubeConfig := os.Getenv(kubeconfigEnvVariable)
```

Comment:
Chris just had a PR this week about a new kubeconfig manager

Reply:
Ok let me check on it.

Reply:
Ok moved the config setup to kubeconfig file from Chris's PR.